tests/test_db.py

```diff
@@ -445,13 +438,38 @@ def test_add_organization(self):
         name = 'Example'
 
         org = db.add_organization(self.trxl, name)
-        self.assertIsInstance(org, Organization)
+        self.assertIsInstance(org, Group)
```

Comment:
Why did you change this test? I think it should keep testing that the object is an organization.